chapters/classes.tex

```diff
@@ -341,21 +341,21 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 \begin{itemize}
 \item
   A variable \lstinline!vc! declared with \lstinline!constant!\indexinline{constant} prefix remains constant during transient analysis, with a value that is unaffected by the initialization problem.
+  This is called a \firstuse{constant}, or \firstuse{constant variable}\index{constant variable}\index{component variability!constant}.
 \item
   A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix remains constant during transient analysis, with a value determined by the initialization problem.
+  This is called a \firstuse{parameter}, or \firstuse{parameter variable}\index{parameter variable}\index{component variability!parameter}.
 \item
-  A \emph{discrete-time} variable \lstinline!vd! has a vanishing time derivative between events.
-  Note that this is not the same as saying that \lstinline!der(vd)=0! almost everywhere,
-  as the derivative is not even defined at the events, and it is not legal
-  to apply \lstinline!der! to discrete-time variables as they are not continuous. During transient analysis the variable
-  can only change its value at event
-  instants (see \cref{events-and-synchronization}).
+  A \emph{discrete-time} variable \lstinline!vd! is a variable that is discrete-valued (that is, not of \lstinline!Real! type) or assigned in a \lstinline!when!-clause.
```

Comment:
Should probably be introduced more similar to constants and parameters, something like this:
```suggestion
  A \firstuse{discrete-time variable}\index{discrete-time variable}\index{component variability!discrete-time} \lstinline!vd! is a variable that is discrete-valued (that is, not of \lstinline!Real! type) or assigned in a \lstinline!when!-clause.
```